site/source/docs/tools_reference/settings_reference.rst

```diff
@@ -1165,6 +1165,8 @@ https://github.com/WebAssembly/exception-handling/blob/main/proposals/exception-
 If false, emit instructions for the standardized exception handling proposal:
 https://github.com/WebAssembly/exception-handling/blob/main/proposals/exception-handling/Exceptions.md
 
+.. note:: Applicable during both linking and compilation
```

Comment:
Maybe this should be "required during both linking and compilation"? Since you have to use it at both times for it to work?

Reply:
This text is auto-generated from ru nning `update_settings_docs.py`:
https://github.com/emscripten-core/emscripten/blob/498047238d0498a6b54e132d2473e866d5324f5c/tools/maint/update_settings_docs.py#L48-L54
https://emscripten.org/docs/tools_reference/settings_reference.html has that message in many places. We can consider changing it if it's not clear, but it should be another PR I guess.